manim/mobject/svg/brace.py

```diff
@@ -51,35 +53,43 @@ def __init__(
         mobject,
         direction=DOWN,
         buff=0.2,
-        width_multiplier=2,
-        max_num_quads=15,
-        min_num_quads=0,
+        sharpness=2,
+        stroke_width=0,
+        fill_opacity=1.0,
         background_stroke_width=0,
+        background_stroke_color=BLACK,
         **kwargs
     ):
-        self.width_multiplier = width_multiplier
-        self.max_num_quads = max_num_quads
-        self.min_num_quads = min_num_quads
+        path_string_template = "m0.01216 0c-0.01152 0-0.01216 6.103e-4 -0.01216 0.01311v0.007762c0.06776 0.122 0.1799 0.1455 0.2307 0.1455h{0}c0.03046 3.899e-4 0.07964 0.00449 0.1246 0.02636 0.0537 0.02695 0.07418 0.05816 0.08648 0.07769 0.001562 0.002538 0.004539 0.002563 0.01098 0.002563 0.006444-2e-8 0.009421-2.47e-5 0.01098-0.002563 0.0123-0.01953 0.03278-0.05074 0.08648-0.07769 0.04491-0.02187 0.09409-0.02597 0.1246-0.02636h{0}c0.05077 0 0.1629-0.02346 0.2307-0.1455v-0.007762c-1.78e-6 -0.0125-6.365e-4 -0.01311-0.01216-0.01311-0.006444-3.919e-8 -0.009348 2.448e-5 -0.01091 0.002563-0.0123 0.01953-0.03278 0.05074-0.08648 0.07769-0.04491 0.02187-0.09416 0.02597-0.1246 0.02636h{1}c-0.04786 0-0.1502 0.02094-0.2185 0.1256-0.06833-0.1046-0.1706-0.1256-0.2185-0.1256h{1}c-0.03046-3.899e-4 -0.07972-0.004491-0.1246-0.02636-0.0537-0.02695-0.07418-0.05816-0.08648-0.07769-0.001562-0.002538-0.004467-0.002563-0.01091-0.002563z"
+        default_min_width = 0.90552
```

Comment:
Why is this `.90552` and not an integer like 1?

Reply:
I scaled the SVG path string so that it is defaults to the size of the previous implementation using Tex.